index.html

```diff
@@ -143,36 +179,35 @@ <h2 class="works-card-heading-large">Tonic</h2>
           </div>
         </li>
         <!-- Card 3 Ends -->
-      
-                <!-- Card 4 starts -->
-                <li class="works-card">
-                  <img src="./assets/card2.png" alt="Recent Work card-2" />
-                  <div class="works-card-main">
-                    <h2 class="works-card-heading-large">Tonic</h2>
-                    <div class="works-card-client">
-                      <p class="paragraph">UBER</p>
-                      <p class="fa-solid fa-circle works-card-client-counter"></p>
-                      <p class="works-card-client-role paragraph">Back End Dev</p>
-                      <p class="fa-solid fa-circle works-card-client-counter"></p>
-                      <p class="works-card-client-year paragraph">2015</p>
-                    </div>
-                    <p class="paragraph works-card-paragraph">
-                      A daily selection of privately personalized reads; no accounts or sign-ups required.
-                    </p>
-                    <ul class="works-card-tags">
-                      <li class="works-card-tag">html</li>
-                      <!-- <li class="works-card-tag">Ruby on rails</li> -->
-                      <li class="works-card-tag">css</li>
-                      <li class="works-card-tag">javascript</li>
-                    </ul>
-                    <button>See Project</button>
-                  </div>
-                </li>
-              
-              </ul>
-            </section>
-            <!--Card works ends-->
-        
+
+        <!-- Card 4 starts -->
+        <li class="works-card">
+          <img src="./assets/card2.png" alt="Recent Work card-2" />
+          <div class="works-card-main">
+            <h2 class="works-card-heading-large">Tonic</h2>
+            <div class="works-card-client">
+              <p class="paragraph">UBER</p>
+              <p class="fa-solid fa-circle works-card-client-counter"></p>
+              <p class="works-card-client-role paragraph">Back End Dev</p>
+              <p class="fa-solid fa-circle works-card-client-counter"></p>
+              <p class="works-card-client-year paragraph">2015</p>
+            </div>
+            <p class="paragraph works-card-paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="works-card-tags">
+              <li class="works-card-tag">html</li>
+              <!-- <li class="works-card-tag">Ruby on rails</li> -->
```

Comment:
Kindly do not leave any unused codes as comments in your code base since it makes your code messy and bulky as the project becomes bigger. :wink: